com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs

```diff
@@ -18,23 +20,51 @@ public override void OnInspectorGUI()
 
             var streamPlayer = target as AlembicStreamPlayer;
             var targetStreamDesc = streamPlayer.StreamDescriptor;
-            var multipleTimeRanges = false;
-            foreach (AlembicStreamPlayer player in targets)
+
+            if (streamPlayer.ExternalReference && !serializedObject.isEditingMultipleObjects)
+            {
+                var initialFilePath = targetStreamDesc != null ? targetStreamDesc.PathToAbc : "";
+                var filePath = initialFilePath;
+                EditorGUILayout.LabelField(new GUIContent("Alembic File"));
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    filePath = EditorGUILayout.DelayedTextField(filePath);
+                    if (GUILayout.Button(new GUIContent("..."), GUILayout.MaxWidth(30)))
+                    {
+                        var path = EditorUtility.OpenFilePanel("Load Alembic File", "", "abc");
+                        if (!string.IsNullOrWhiteSpace(path))
+                            filePath = path;
+                    }
+                }
+
+                if (filePath != initialFilePath)
+                {
+                    Undo.RegisterFullObjectHierarchyUndo(streamPlayer.gameObject, "Load Alembic File");
+                    streamPlayer.LoadFromFile(filePath);
+                }
+
+                EditorGUILayout.HelpBox(
+                    "Alembic File is streamed. Selecting a new file will rebuild the GameObject or Prefab hierarchy.",
```

Comment:
Suggestion:
`You have set up your project to stream the Alembic file. If you select a new file, Unity rebuilds the GameObject or Prefab hierarchy.`

Reply:
This is not even true anymore...
Will remove for now.